merchant/src/catalog/backend.rs

```diff
@@ -425,12 +558,20 @@ async fn increase_item_variation_units(
     Ok(())
 }
 
+fn make_id_by_alias(id: &str) -> String {
+    return format!("{}", id);
```

Comment:
```suggestion
    id.into()
```
No need for the explicit return, this function doesn't look very useful IMO

Reply:
Look 👁️, I forget to remove this!